tests/bot/exts/info/test_information.py

```diff
@@ -465,11 +393,6 @@ def test_create_user_embed_basic_information_outside_of_moderation_channels(self
             embed.fields[1].value
         )
 
-        self.assertEqual(
-            "basic infractions info",
-            embed.fields[3].value
-        )
```

Comment:
I don't see why this would fail considering you never touched the infraction info code.

Reply:
It was due to the fact that I have removed 3rd field which had informations about user's status, I have since added the test back and changed the index to `2` in 764f35f as currently, infractions will be in 3rd field and not 4th